src/components/Firebase.js

```diff
@@ -1,11 +1,30 @@
 import firebase from 'firebase'
 const config = {
-    apiKey: "AIzaSyDtEWBHELCyBKOww5fRNdnf3PXDWye5TWs",
-    authDomain: "sugar-donut.firebaseapp.com",
-    databaseURL: "https://sugar-donut.firebaseio.com",
-    projectId: "sugar-donut",
-    storageBucket: "sugar-donut.appspot.com",
-    messagingSenderId: "1024720151938"
-};
-firebase.initializeApp(config);
-export default firebase;
\ No newline at end of file
+  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
+  authDomain: process.env.REACT_APP_FIREBASE_AUTH_DOMAIN,
+  databaseURL: process.env.REACT_APP_FIREBASE_DATABASE_URL,
+  projectId: process.env.REACT_APP_FIREBASE_PROJECT_ID,
+  storageBucket: process.env.REACT_APP_FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGING_SENDER_ID
+}
+
+let missingVars = []
+let firebaseApp = null
+
+Object.keys(config).forEach((key) => {
+  let val = config[key]
+  console.log(val)
```

Comment:
Will this log all the env vars to the browser console? Probably don't want to do that!